docs/develop/power-automate-integration.md

```diff
@@ -0,0 +1,127 @@
+
+2. The basic types of `string`, `number`, `boolean`, `void`, and `undefined` are supported.
+
+3. Union types used as return types follow the same restrictions as they do when used as script parameters.
+
+4. Array types are allowed if they are of type `string`, `number`, or `boolean`. They are also allowed if the type is a supported union or supported literal type.
+
+5. Object types used as return types follow the same restrictions as they do when used as script parameters.
+
+6. Implicit typing is supported, though it must follow the same rules as a defined type.
+
+## Absolute references
+
+Power Automate runs your script in the chosen Excel workbook on your behalf. The workbook might be closed when this happens. Any API that relies on the user's current state, such as `WorksheetCollection.getActiveWorksheet`, will fail when ran through Power Automate. When designing your scripts, be sure to use absolute references for worksheets and ranges.
+
+The following functions will throw an error and fail when called from a script in a Power Automate flow.
+
+- `Chart.activate`
+- `Range.select`
+- `Workbook.getActiveCell`
+- `Workbook.getActiveChart`
+- `Workbook.getActiveChartOrNullObject`
+- `Workbook.getActiveSlicer`
+- `Workbook.getActiveSlicerOrNullObject`
+- `Workbook.getSelectedRange`
+- `Workbook.getSelectedRanges`
+- `Worksheet.activate`
+- `WorksheetCollection.getActiveWorksheet`
+
+## Example
+
+Our example flow is triggered whenever a [GitHub](https://github.com/) issue is assigned to you. The issue is recorded in a table in an Excel workbook. If there are five or more issues in that table, the flow sends an email reminder.
+
+![The example flow as shown in the Power Automate flow editor.](../images/power-automate-parameter-return-sample.png)
+
+The script takes in the issue ID and issue title as parameters. It returns the number of rows in the issue table.
+
+```TypeScript
+async function main(
+  context: Excel.RequestContext,
+  issueId: string,
+  issueTitle: string
+  ): Promise<number> {
+  // Get the "GitHub" worksheet.
+  let worksheet = context.workbook.worksheets.getItem("GitHub");
+
+  // Get the first table in this worksheet, which contains the table of GitHub issues.
+  let issueTable = worksheet.tables.getItemAt(0);
+  
+  // Add the issue ID and issue title as a row.
+  issueTable.rows.add(-1, [[issueId, issueTitle]]);
+
+  // Return the number of rows in the table, which represents how many issues are assigned to this user.
+  return issueTable.rows.count;
+}
+```
+
+## See also
+
+- [Run Office Scripts in Excel on the web with Power Automate](../tutorials/excel-power-automate-manual.md)
+- [Integrate Office Scripts into automated Power Automate flows](../tutorials/excel-power-automate-trigger.md)
+- [Scripting fundamentals for Office Scripts in Excel on the web](scripting-fundamentals.md)
+- [Get started with Power Automate](https://docs.microsoft.com/power-automate/getting-started)
```

Comment:
Nit: the URL for any docs.microsoft.com link doesn't actually need the `https://docs.microsoft.com` portion -- e.g., this link can just be changed to -> /power-automate/getting-started